cmd/proxy.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/gravitl/netclient/functions"
+	"github.com/spf13/cobra"
+)
+
+// proxyCmd represents the proxy command
+var proxyCmd = &cobra.Command{
+	Use:   "proxy [ on | off ]",
+	Short: "proxy on/off",
+	Long:  `switches proxy on/off`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println("\nmissing status [ on | off ] argument in the command")
+			return
+		}
+		err := functions.ChangeProxyStatus(getStatus(args[0]))
+		if err != nil {
+			fmt.Println(err.Error())
+		}
+	},
+}
+
+func getStatus(status string) bool {
+	return status == "on"
```

Comment:
I think there should be explicit checks for on/off, otherwise one could say `netclient proxy dfawsiuhfgi` and it would turn off.